components/gravityforms/class-gf-convertkit.php

```diff
@@ -210,6 +234,50 @@ public function get_custom_fields() {
 		return $fields;
 	}
 
+	/**
+	 * Get ConvertKit Tags from the API to be used for the dynamic field map
+	 *
+	 * @return array
+	 */
+	public function get_tags() {
+
+		$path         = 'tags';
+		$query_args   = array();
+		$request_body = null;
+		$request_args = array();
+		$tags       = array(
+			array(
+				'value' => null,
+				'label' => 'Choose a Tag',
```

Comment:
Need to localize 'Choose a Tag' string with e.g. `__( 'Choose a Tag', 'convertkit' )` (although I'm aware much of the Plugin seems to be missing full localization strings, which is a fault on our part).

Reply:
Yeah I was just following along with most of the patterns I'm seeing. Definitely needs localization. :)